src/content/1.7/themes/distribution/exporting.md

```diff
@@ -10,8 +10,8 @@ There is no longer any theme data in the database with PrestaShop 1.7. Hence a t
 it's on the disk.
 
 If you want to theme to appears in the backoffice, it's simply have to contain a `config/theme.yml` file.
```

Comment:
```suggestion
If you want the theme to appears in the backoffice, it simply has to contain a `config/theme.yml` file.
```

I've some doubt about my english, but it looks like there is some problems there too, I know it's not related to your PR but maybe we can improve this at the same time regarding the fact that you open some PRs to improve typos :)